### PR TITLE
AA-762:View to configure location and roles for a user

### PR DESCRIPTION
**Pull request by kimaina**
test build: https://test1.ampath.or.ke:8443/test-builds/AA-762/

### CODE REVIEW

File: app/scripts/administration/controllers/administration.controller.js [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+   * @description
+   * # MainCtrl
+   * Controller of the ngAmrsApp
+   */
+  angular
+    .module('app.administration')
+    .controller('AdministrationCtrl', AdministrationCtrl);
+  AdministrationCtrl.$nject = ['$rootScope', '$scope'];
+
+  function AdministrationCtrl($rootScope, $scope) {
```
Comment (mhawila) [inline app/scripts/administration/controllers/administration.controller.js:17]: @kimaina  this controller looks like is completely useless!

Comment (kimaina) [inline app/scripts/administration/controllers/administration.controller.js:17]: it is the controller that holds the locationAuthorization directive.

Comment (kimaina) [inline app/scripts/administration/controllers/administration.controller.js:17]: I followed patient dashboard design, where we have a main controller that serves several other directives. So we expect in future we will have several other sub modules in admin dashboard module eg broadcast feature e.t.c

File: app/scripts/administration/services/location-authorization.service.js [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+        userProperties['grantAccessToLocation[*]'] = '*';
+      } else if (selectedLocations.locations.length > 0) {
+        for (var i = 0; i < selectedLocations.locations.length; i++) {
+          userProperties['grantAccessToLocation[' + i + ']'] = selectedLocations.locations[i].uuId();
+        }
+      }
+
+      //finally create payload and add uuid to identify selected user
+      payload.uuid = selectedUser.selected.uuid;
+      payload.userProperties=userProperties;
```
Comment (mhawila) [inline app/scripts/administration/services/location-authorization.service.js]: What is the purpose of this statement? Why do you need to resubmit the already existing user properties?

**Pull request merged**